scripts/archivenode.bash

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+git checkout v1.0.1
```

Comment:
```suggestion
git checkout v3.2.0
```

No reason to ever be using anything older, since its all state compatible!

Reply:
are you certain of this?

I've found it to explode unless following this pattern